.travis.yml

```diff
@@ -1,25 +1,51 @@
-dist: xenial
+dist: bionic
 os: linux
 language: node_js
 services:
-- docker
+  - docker
 node_js:
-- 12.18.2
-env:
-  jobs:
```

Comment:
this env matrix enables parallel jobs, are you intentionally disabling that?

Reply:
Yes, it is disabled as we need to run 2 languages: node and ruby.  I'm probably missing something; the Travis documentation is pretty bad at showing useful examples.